### DISCUSSION TITLE
SED Discussion 26.06.2023

### DISCUSSION THREAD

**Discussion by kutnyakhov** [OpenCOMPES/sed#135] [General]
Topics SED meeting:
- next meeting date after the summer break - potential date - 15.08.2023 afternoon, e.g. 13:00;
- one more setup support is now possible with sed - WESPE-double ToF (Themis1000) spectrometer with DLDs.
Data have identical structure to hextof, but two different DLDs.
Currently, we are using a flash loader with two different config files to read data and create two different dataframes
During data processing, we have to do an intermediate step of converting data with the help of SimIon.
For this saving and loading parquet was implemented;
- we hope to use new WESPE support during the following two beamtimes at FLASH starting next week;
- The date for the next meeting is chosen right before the start of the first commissioning beamtime @EuXFEL,
just in case there is some urgent question about how to read data or so;
- Workflow manager is slowly moving, but probably will be on standby as Steinn going on a long vacation;

**Comment by zain-sohail** [comment]
Steinn said he will push the latest workflow manager branch so we can work on it further, if possible.
